### ISSUE TITLE
Is this maintained?

### ISSUE THREAD

**Issue by ChildishGiant** [rxi/lite#281]
There are lots of open pull requests and there's been no activity for 6 months? Is there a community fork with more development going on?

**Comment by thechidambaram** [comment]
I am not aware if this is maintained by rxi anymore, but there is a fork, [lite-xl](https://github.com/franko/lite-xl) that is still maintained.

**Comment by AKonia** [comment]
As i understand it is **complete**, because it is minimalistic code editor with caching-software rendering. If you needs more features - you can install some plugins or write them or take a look for a forks(but forks are doing things more complicated)

**Comment by ThatGuyMike7** [comment]
Yeah I think lite is complete, and it's great that way. If you want more features, you can download a fork or create your own.  The codebase is small and simple, so you can easily extend it to fit your needs.

**Comment by nandgator** [comment]
Maybe someone can pin this issue, or put. status as `complete` in the readme?
